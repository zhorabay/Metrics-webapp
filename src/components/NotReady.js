import { Link } from 'react-router-dom';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faAngleLeft } from '@fortawesome/free-solid-svg-icons';
import '../styles/NotReady.css';

const NotReady = () => {
  return (
    <>
      <h2>This page will be updated later</h2>
      <Link className="back1" to="/">
        <FontAwesomeIcon className="back-icon1" icon={faAngleLeft} />
      </Link>
    </>
  );
};

export default NotReady;
