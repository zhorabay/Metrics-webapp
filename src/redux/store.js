import { configureStore } from '@reduxjs/toolkit';
import logger from 'redux-logger';
import comicsReducer from './comicsSlice';

const store = configureStore({
  reducer: {
    comics: comicsReducer,
  },
  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
});

export default store;
