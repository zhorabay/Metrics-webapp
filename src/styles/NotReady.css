* {
  margin: 0;
  padding: 0;
}

.back-icon1 {
  color: white;
  border: 2px solid #757373;
  background-color: rgb(50, 50, 50);
  border-radius: 100px;
  padding: 8px 10px;
}
