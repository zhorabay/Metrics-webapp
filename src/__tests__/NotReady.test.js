import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/store';
import NotReady from '../components/NotReady';

test('NotReady component matches snapshot', () => {
  const { asFragment } = render(
    <Provider store={store}>
      <Router>
        <NotReady />
      </Router>
    </Provider>,
  );
  expect(asFragment()).toMatchSnapshot();
});
