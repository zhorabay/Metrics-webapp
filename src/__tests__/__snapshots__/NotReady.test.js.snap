// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotReady component matches snapshot 1`] = `
<DocumentFragment>
  <h2>
    This page will be updated later
  </h2>
  <a
    class="back1"
    href="/"
  >
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-angle-left back-icon1"
      data-icon="angle-left"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 320 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M41.4 233.4c-12.5 12.5-12.5 32.8 0 45.3l160 160c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L109.3 256 246.6 118.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0l-160 160z"
        fill="currentColor"
      />
    </svg>
  </a>
</DocumentFragment>
`;
